src/Solvers/sparse_preconditioners.jl

```diff
@@ -174,3 +176,31 @@ function sparse_inverse_preconditioner(A::AbstractMatrix; ε, nzrel)
    Minv = arch_sparse_matrix(architecture(A), Minv_cpu)
    return SparseInversePreconditioner(Minv)
 end
+
+multigrid_preconditioner(A::AbstractMatrix) = aspreconditioner(create_multilevel(RugeStubenAMG(), A))
```

Comment:
Would something like that work? Defaults to `RugeStubenAMG()` but allows for one to use `SmoothedAggregationAMG()` if they'd like?
 
```suggestion
multigrid_preconditioner(A::AbstractMatrix; method = RugeStubenAMG()) = aspreconditioner(create_multilevel(method, A))
```